package.json

```diff
@@ -1,56 +1,51 @@
 {
   "name": "eslint-config-molindo",
-  "version": "7.0.0",
+  "version": "8.0.0",
   "description": "Molindo ESLint config that implements our styleguide and helps to catch errors.",
   "repository": "https://github.com/molindo/eslint-config-molindo",
   "author": "Jan Amann <jam@molindo.at>",
+  "type": "module",
   "license": "MIT",
+  "main": "src/index.js",
   "keywords": [
     "eslint",
     "eslintconfig"
   ],
   "scripts": {
-    "lint": "eslint .",
-    "test": "jest",
-    "prepublish": "npm run lint && npm run test"
+    "lint": "eslint src",
+    "prepublish": "npm run lint"
   },
   "files": [
-    "setupPlugins.js",
-    "javascript.js",
-    "typescript.js",
-    "config.js",
-    "css-modules.js",
-    "tailwind.js",
-    "react.js",
-    "jest.js",
-    "cypress.js",
+    "src",
+    ".prettierrc.json",
     "tsconfig.json"
   ],
   "devDependencies": {
-    "eslint": "^8.45.0",
-    "jest": "^29.6.2",
-    "typescript": "5.1.6"
+    "eslint": "^9.0.0",
+    "prettier": "^3.3.0",
+    "tailwindcss": "^3.4.13",
+    "typescript": "^5.1.6"
   },
   "peerDependencies": {
-    "eslint": "^8.0.0"
+    "eslint": "^9.0.0"
```

Comment:
Prettier is not a strict peer dependency technically, but strongly recommended (and kind of assumed). Not listing it here gives flexibility to consumers though, e.g. to use alternative formatters like Biome or Oxc.